tests/test_io.py

```diff
@@ -1,11 +1,40 @@
 import tempfile
 from io import BufferedReader, BytesIO, StringIO, TextIOWrapper
 
+import numpy as np
 import pytest
 
 from mirdata import io
 
 
+def test_load_midi():
+    midi_file = (
+        "tests/resources/mir_datasets/maestro/2018/"
+        + "MIDI-Unprocessed_Chamber3_MID--AUDIO_10_R3_2018_wav--1.midi"
+    )
+    midi = io.load_midi(midi_file)
+    assert len(midi.instruments) == 1
+    assert len(midi.instruments[0].notes) == 4197
+
+
+def test_load_notes_from_midi():
```

Comment:
do you also test note value?

Reply:
what do you mean? In the lines below I test the values, no?

Reply:
yes! you are right! sorry!